File: DSA/PranyaGupta_500126032/Day2Question2.py
class Solution:
    def FirstAndLastElement(self, arr):
        first = arr[0]
        last = arr[-1]
        return f"First: {first}, Last: {last}"
def main():
    user_input = input("Enter elements of the array (with space in between): ")
    arr = list(map(int, user_input.strip().split()))
    sol = Solution()
    result = sol.FirstAndLastElement(arr)
    print(result)
if __name__ == "__main__":
    main()

File: DSA/Shreya_Goyal_590015491/Day_2_Question_2.c
#include <stdio.h>

int main() {
    int arr[5];
    int i;
        
    printf("Enter 5 elements:\n");
    for(i=0; i<5; i++){
     scanf("%d", &arr[i]);
    }

printf("First:%d , Last:%d \n",arr[0],arr[4]);

return 0;
}